File: containsDuplicate.java
import java.util.HashSet;

class Solution {
    public boolean containsDuplicate(int[] nums) {
        HashSet<Integer> set = new HashSet<Integer>();
        for (int i = 0; i < nums.length; i++) {
            if (set.contains(nums[i])) {
                return true;
            }
            set.add(nums[i]);
        }
        return false;
    }

    public static void main(String[] args) {
        Solution solution = new Solution();
        int[] nums = {1, 2, 3, 4,5,6,7,1};
        boolean result = solution.containsDuplicate(nums);
        System.out.println(result);  // This should print "true"
    }
}

File: containsDuplicate.js
function containsDuplicate(nums) {
    const set = new Set();
    for (let i = 0; i < nums.length; i++) {
        if (set.has(nums[i])) {
            return true;
        }
        set.add(nums[i]);
    }
    return false;
}

const nums = [1, 2, 3, 4];
console.log(containsDuplicate(nums));  // This should print "true"